en/middleware/micrortps.md

```diff
@@ -215,13 +225,11 @@ Check the **Building the `px4_ros_com` package** for details about the build str
 
 ## Building the `px4_ros_com` package
 
-As aforementioned, the `px4_ros_com` comes bundled with two branches, where one links the ROS2 with the PX4-agent/client bridge and the other allows, through `ros1_bridge`, to link the ROS framework with the ROS2 framework through the same set of messages. 
-Therefore, both branches need to be cloned separately so the build process can happen correctly. 
-Before that can happen, one requires to install and setup both ROS2 and ROS environments on its machine.
+Before building `px4_ros_com` you will first need to clone the PX4 Firmware repository
+(this is normally placed at the same tree level as the ROS workspaces).
+Then install and setup both ROS2 and ROS environments on your development machine
```

Comment:
@TSC21 Do we need to install the ros1 branch if we just want to use ROS2?

Reply:
No we don't need the ROS1 branch if you are just going to use the ROS2 middleware.